sw/device/silicon_creator/lib/sigverify_tests/sigverify_set_testvectors.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```

Comment:
just a note: I think most of this functionality will be available in the opentitantool, we should replace the code here with calls to opentitantool when it's ready.